__tests__/app.test.js

```diff
@@ -116,156 +31,255 @@ describe('app routes', () => {
       return client.end(done);
     });
 
-    // test('returns animals', async() => {
+    // test('POST', async() => {
```

Comment:
I saw your screenshots, but for next time, you will want to make sure all tests are uncommented and that they can pass CI.  CI automatically runs the tests so will tell us if those tests pass or not.   Let me know if you have questions on this.